src/components/PageElements/Elements/ApplicantResponseElement.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Grid, Icon, Label } from 'semantic-ui-react'
+import { SummaryViewWrapper } from '../../../formElementPlugins'
+import { SummaryViewWrapperProps } from '../../../formElementPlugins/types'
+import strings from '../../../utils/constants'
+import getSimplifiedTimeDifference from '../../../utils/dateAndTime/getSimplifiedTimeDifference'
+import { ResponseElementProps } from '../../../utils/types'
+
+type ApplicantResponseElementProps = ResponseElementProps & SummaryViewWrapperProps
+const ApplicantResponseElement: React.FC<ApplicantResponseElementProps> = ({
+  applicationResponse,
+  isResponseUpdated = false,
+  children,
+  ...summaryViewProps
+}) => {
+  const backgroudColour = isResponseUpdated ? 'changable-background' : ''
+  // TODO: Fix tiny margin showing on left and right on grid.item
```

Comment:
If possible, you should change the background of the parent element, the `summary-page-element`, and then you won't have to deal with this margin discrepancy. If you add the `changeable-background` (dynamically) class to the same div as `summary-page-element`, the blue colour should over-ride the white (assuming it's later in the CSS file.)

Reply:
Hi @CarlosNZ I tried to do that but it didn't work as simple as this. I'll probably need some help with that, if we can have a look together in about 15 minutes?

Reply:
I had a quick play last night, and I thought it was working ok -- let's take a quick look when you're ready